src/main/java/com/gtnewhorizons/angelica/compat/mojang/Axis.java

```diff
@@ -9,10 +9,9 @@ public enum Axis {
 
     public static Axis fromDirection(ModelQuadFacing dir) {
         return switch (dir) {
-            case DOWN, UP -> Y;
+            case UP, DOWN, UNASSIGNED -> Y;
```

Comment:
Why does UNASSIGNED have an axis? This seems incorrect.

Reply:
Avoids an NPE on launch (for some reason, a quad is made without an axis) and mirrors Vintagium behavior of defaulting to DOWN.

Reply:
The unaligned quad probably comes from a tall grass model or something similar. 1.16.5 assigns UP as the default direction to baked quads since UNASSIGNED is not an option in vanilla. Maybe mirroring this in `Quad` would be more accurate.